frontend/src/components/form/Form.vue

```diff
@@ -26,6 +27,7 @@
 
 @media screen and (max-width: 512px) {
   .form {
+    min-width: 100%;
```

Comment:
```suggestion
    width: 100vw;
```
뷰포트 너비를 이용하는 게 좋을 것 같네요!